saleor/setting/models.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+from django.utils.translation import pgettext_lazy
+from django.utils.encoding import python_2_unicode_compatible
+
+
+INTEGER = 'i'
+STRING = 's'
+BOOLEAN = 'b'
+
+
+@python_2_unicode_compatible
+class Setting(models.Model):
```

Comment:
I'd much rather call this `Site` and give it all of the existing properties of `django.contrib.sites.models.Site`. We could then have one object instance per site (with actual settings living in either a JSON field or an hstore field) and use things like site name in our templates.